Formula/a/aws-auth.rb

```diff
@@ -16,6 +16,8 @@ class AwsAuth < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "a60b04e342a6f7740665def07e9a7851d7214d47e1da66641e4b6692c7b67067"
   end
 
+  disable! date: "2025-03-14", because: :repo_removed
```

Comment:
should we deprecate first as the bottle still works?

Reply:
I wasn't sure what was best but I'll update this to deprecate instead in the morning.

Reply:
we had such cases before, yeah, deprecate is a better route imo